modules/table/src/test/java/org/apache/ignite/internal/table/distributed/IndexBaseTest.java

```diff
@@ -129,7 +130,8 @@ void setUp(@InjectConfiguration DataStorageConfiguration dsCfg) {
                 PARTITION_ID,
                 new TestPartitionDataStorage(storage),
                 DummyInternalTableImpl.createTableIndexStoragesSupplier(indexes),
-                dsCfg
+                dsCfg,
+                new PendingComparableValuesTracker<>(HybridTimestamp.MAX_VALUE)
```

Comment:
Same here, it's not obvious why you've chosen the max value.
And another question second time - are you sure that adding all this stuff to storage update handler is a good idea?

Reply:
So that when inserting into a table, the GC always worked for us.

> And another question second time - are you sure that adding all this stuff to storage update handler is a good idea?
answered above.